app/components/NoticeTypeCheckboxes/NoticeTypeCheckboxes.tsx

```diff
@@ -229,6 +229,14 @@ export function NoticeTypeCheckboxes({
   if (useFeature('SVOM_QUICKSTART')) {
     JsonNoticeTypes.SVOM = ['gcn.notices.svom']
     JsonNoticeTypeLinks.SVOM = '/missions/svom'
```

Comment:
Since there currently aren't any SVOM JSON notices, does removing this feature flag also add SVOM to the JSON list?

Reply:
Yea, is there a different feature flag this should be behind (if one at all?)

Reply:
Feature flag `SVOM` is used elsewhere, like on the mission page.  Let's put the JSON under the flag `SVOM_JSON`, so we can track it separately from the VOEvent, mission page, and announcement that we're very close to releasing.